src/styles/utils/variables.scss

```
@@ -0,0 +1,9 @@
+$c-blue: #00acdc;
+$c-main-accent: #060b35;
+$c-secondary: #616070;
+$c-white: #fff;
+$c-hover-title: #34568b;
+
+$mobile-min-with: 488px;
```

As I understand it, the `width` is meant?
If so, it should be fixed everywhere.
```suggestion
$mobile-min-width: 488px;
```